src/app/shared/pipes/groupLink.ts

```diff
@@ -0,0 +1,15 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { UrlCommand } from '../helpers/url';
+import { GroupRoute } from '../routing/group-route';
+import { urlArrayForGroup } from '../routing/group-route';
+
+/**
+ * Functions using item route should always be preferred to raw item.
+ * Using raw item means further requests will be required to fetch path and attempt information.
+ */
+@Pipe({ name: 'groupLink', pure: true })
+export class GroupLinkPipe implements PipeTransform {
+  transform(route: GroupRoute, options?: { page?: 'edit' | 'details'; isUser?: boolean }): UrlCommand {
```

Comment:
`isUser` is handled, I wonder if you were not reviewing [this old commit](https://github.com/France-ioi/AlgoreaFrontend/pull/625/commits/4b3d074f2e8b6fdeab86743cdc1fe27dbca6cfea) instead of the latest one

I'll keep the `userLink` pipe in mind for later PR :+1: 

The way to determine if a group is a user is not the login, this here is a `WatchedGroup` object, not a group
About the group `type: "Class" "Team" "Club" "Friends" "Other" "Session" "Base"`, which ones indicate a user ?
As letting the pipe do that, I'd rather not since it is a `WatchedGroup` and not a group, it would make the pipe not reusable

Reply:
Fixed